### PR TITLE
(fix) Deleting a condition successfully should update the conditions table

### PR DESCRIPTION
**Pull request by Jexsie**
## Requirements

- [x] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-dev.docs.openmrs.org/#/getting_started/contributing?id=your-pr-title-should-indicate-the-type-of-change-it-is) label. See existing PR titles for inspiration.
- [x] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://zeroheight.com/23a080e38/p/880723-introduction).
- [x] My work includes tests or is validated by existing tests.

## Summary

- Passes the required `patientUuid` to the delete condition modal for handling mutation
## Screencasts
[Screencast from 2023-06-28 05-42-29.webm](https://github.com/openmrs/openmrs-esm-patient-chart/assets/104269786/3bcb9c01-2f09-4033-a4e3-695c271576e0)

### CODE REVIEW

File: packages/esm-patient-notes-app/src/notes/visit-notes-form.scss [hunk 1]
```diff
@@ -70,7 +70,6 @@
   display: grid;
   grid-template-columns: 50% 10% 1fr;
   margin: 0.5rem 1rem;
-  width: 100%;
```
Comment (Jexsie) [inline packages/esm-patient-notes-app/src/notes/visit-notes-form.scss:73]: missed this out when Removing side scrolling in allergies, start visit and visit notes forms

File: packages/esm-patient-conditions-app/src/conditions/conditions-action-menu.component.tsx [hunk 2]
```diff
@@ -28,6 +29,7 @@ export const ConditionsActionMenu = ({ condition }: conditionsActionMenuProps) =
     const dispose = showModal('condition-delete-confirmation-dialog', {
       closeDeleteModal: () => dispose(),
       conditionId,
+      patientUuid,
```
[the remark this answers is not in the document]
Comment (Jexsie) [reply packages/esm-patient-conditions-app/src/conditions/conditions-action-menu.component.tsx:32]: yes, if you look closely, we were not passing it while launching the modal yet it's needed by the `useConditions` hook.

**Comment by hadijahkyampeire** [comment]
@Jexsie could you attach some recording.

**Comment by Jexsie** [comment]
> @Jexsie could you attach some recording.

Updated. Thanks @hadijahkyampeire

File: packages/esm-patient-conditions-app/src/conditions/conditions-action-menu.component.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (denniskigen) [reply packages/esm-patient-conditions-app/src/conditions/conditions-action-menu.component.tsx:32]: Thanks for clarifying. I'll review once https://github.com/openmrs/openmrs-esm-patient-chart/pull/1249 gets merged in.

**Review by denniskigen — APPROVED** [review]
LGTM. Thanks for the fix, @Jexsie.

**Pull request merged**